apps/api/src/app/shared/helpers/e2e/sdk/e2e-sdk.helper.ts

```diff
@@ -0,0 +1,32 @@
+import { Novu } from '@novu/api';
+// eslint-disable-next-line no-restricted-imports
+import { NovuCore } from '@novu/api/core';
+
+import { UserSession } from '@novu/testing';
+// eslint-disable-next-line no-restricted-imports
+import { SDKError } from '@novu/api/models/errors/sdkerror';
+import { expect } from 'chai';
+
+export function initNovuClassSdk(session: UserSession): Novu {
+  return new Novu({ apiKey: session.apiKey, serverURL: session.serverUrl, debugLogger: console });
+}
+export function initNovuFunctionSdk(session: UserSession): NovuCore {
+  return new NovuCore({ apiKey: session.apiKey, serverURL: session.serverUrl, debugLogger: console });
```

Comment:
What is the difference between Novu and NovuCore in the generated SDK?

Reply:
Novu is the Client and the base line for all of the regular api's so it's (new Novu()).organizations.create() 
novu core is the first parameter in createOrganization(new NovuCore(), dto)

Reply:
This was resolved offline. We will chat with the Speakeasy.